src/web/app/components/session-edit-form/session-edit-form.component.scss

```diff
@@ -42,3 +42,28 @@
   cursor: not-allowed;
   color: gray;
 }
+
+.create-new-dropdown-div {
+  display: flex;
+}
+
+.session-heading-text {
+  display: flex;
+  text-align: left;
```

Comment:
i don't think this is necessary as you've already set the left margin and padding to 0.